docs/docs/deployment/overview.md

```diff
@@ -43,42 +44,6 @@ postgres:
   port: 5432
   dbname: postgres
   params: sslmode=disable
-
-poolingConfig:
-  maxWaitTimeForTrace: 10m
-  retryDelay: 5s
-
-googleAnalytics:
-  enabled: true
-
-demo:
-  enabled: []
-
-experimentalFeatures: []
-
-telemetry:
-  dataStores:
-    jaeger:
-      type: jaeger
-      jaeger:
-        endpoint: jaeger:16685
-        tls:
-          insecure: true
-
-  exporters:
-    collector:
-      serviceName: tracetest
-      sampling: 100 # 100%
-      exporter:
-        type: collector
-        collector:
-          endpoint: otel-collector:4317
-
-server:
-  telemetry:
-    dataStore: jaeger
-    exporter: collector
-    applicationExporter: collector
 ```
 
```

Comment:
Should we include a mention of the `tracetest-provision.yaml` file here?
@jorgeepc @xoscar

Reply:
I think we should have that as a separate PR focused on explaining the provisioning file and have this cleaned up asap so users are not confused.